### PR TITLE
use isFeatureEnabled for security features check

### PR DESCRIPTION
**Pull request by jlipps**
https://github.com/appium/appium-base-driver/pull/328 needs to be published before this will work

### CODE REVIEW

File: lib/commands/logging.js [hunk 1]
```diff
@@ -3,6 +3,8 @@ import IOSLog from '../device-log/ios-log';
 import IOSCrashLog from '../device-log/ios-crash-log';
 import logger from '../logger';
 
+const SERVER_LOGS_FEAT_NAME = 'get_server_logs';
```
Comment (mykola-mokhnach) [inline lib/commands/logging.js]: how about `GET_SERVER_LOGS_FEATURE` ?

File: lib/commands/logging.js [hunk 2]
```diff
@@ -38,9 +40,9 @@ extensions.supportedLogTypes = {
   server: {
     description: 'Appium server logs',
     getter: (self) => {
-      if (!self.relaxedSecurityEnabled) {
-        throw new Error('Appium server must have relaxed security flag set ' +
-                        'in order to retrieve server logs');
+      if (!self.isFeatureEnabled(SERVER_LOGS_FEAT_NAME)) {
+        throw new Error(`Security level does not allow feature '${SERVER_LOGS_FEAT_NAME}'. ` +
+                        `See Appium documentation for allowing insecure features.`);
```
Comment (imurchie) [inline lib/commands/logging.js]: Can this link to the actual docs?

Comment (jlipps) [reply lib/commands/logging.js]: Yes, will work on actual docs. Also will write a function that simply takes a feature name and description and generates this error message, so they all have the same format.

Comment (imurchie) [reply lib/commands/logging.js]: That sounds great!

Comment (jlipps) [reply lib/commands/logging.js]: `ensureFeatureEnabled` links to actual docs.

**Pull request merged**